lib/components/JsonSchema/json-schema.ts

```diff
@@ -55,6 +56,9 @@ export class JsonSchema extends BaseSearchableComponent implements OnInit {
 
   selectDescendantByIdx(idx) {
     this.selectDescendant(this.descendants[idx]);
+    if (this.descendants[idx]) {
+      this._schemaChanger.announceDescendantChange(idx, this.descendants[idx].name, this.isRequestSchema ? true : false, this.responseCode);
```

Comment:
`this.isRequestSchema` is already a boolean, so you could get rid of the ternary operator.